src/core_plugins/console/public/src/directives/help.html

```diff
@@ -1,36 +1,42 @@
-<h3>Help</h3>
+<div class="localDropdownContent localDropdownSection">
```

Comment:
I think you're right. We should definitely figure out a way to make these two classes more obvious to use. Let me think about this and try to come up with a solution that doesn't even require guidance. Thanks man.